JULIERAJ/KIDS-FIRST-Front
# Feat: Edit kid's profile

One Line Description
Implemented Edit Kid's Profile feature, enabling users to edit profiles.

Requirements
- Modify the KidForm component to support both "add" and "edit" modes.
- Follow the design provided in Figma.
- Connect frontend with the backend API for updating kid profiles.

Notes
- The frontend implementation connects to the backend API for editing and deleting kids' profiles.
- Please ensure the backend branch [fix/99/edit-kids-profile](https://github.com/JULIERAJ/KIDS-FIRST-Back/tree/fix/99/edit-kids-profile) in the [KIDS-FIRST-Back repository] is used for testing.


Test Steps
- Navigate to the kid's profile page in the app.
- Click "Edit" on an existing kid's profile to load the KidForm in edit mode.
- Modify fields and save to update the profile.
- Verify successful updates in the database.
- Test adding a new kid profile in "add" mode.
- Ensure the backend branch is checked out and running during testing [fix/99/edit-kids-profile].

UI/UX Outcome
![image](https://github.com/user-attachments/assets/b74b8ddc-b323-4812-8154-67ade4eab762)

Checklist
